bin/current_live_environment

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import os
+import boto3
+
+RUNTIME_ERROR_MESSAGE = """Ensure all environment variables are defined:
+AWS_ACCESS_KEY_ID,
+AWS_SECRET_ACCESS_KEY,
+TARGET_DEPLOY_DOMAIN"""
+
+class GetCurrentLiveEnvironment:
+    def __init__(self, target_domain):
+        self.target_domain = target_domain
+        self.client = boto3.client('route53',
+                                   aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
```

Comment:
I _think_ you can ditch this, and boto will look this stuff up behind the scenes based on ENV vars. And if the ENV vars don't exist, it blows up and complains.

Reply:
So you should just be able to use `boto3.client('route53')`